api/v1/api.pb.go

```diff
@@ -0,0 +1,988 @@
+// Code generated by protoc-gen-go.
+// source: api.proto
+// DO NOT EDIT!
```

Comment:
Since this is a generated file, shouldn't it be removed from the git repository and add calls to `protoc-gen-go` in some makefiles?